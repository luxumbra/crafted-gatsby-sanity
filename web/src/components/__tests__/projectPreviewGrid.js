import React from 'react'
import renderer from 'react-test-renderer'

import ProjectPreviewGrid from '../project-preview-grid'

describe('ProjectPreviewGrid', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <ProjectPreviewGrid
          title="Latest projects"
          browseMoreHref="/projects/"
        />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
