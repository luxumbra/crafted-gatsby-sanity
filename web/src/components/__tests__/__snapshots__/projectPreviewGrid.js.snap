// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectPreviewGrid renders correctly 1`] = `
<div
  className="root"
>
  <div
    className="browseMoreNav"
  >
    <a
      href="/projects/"
    >
      Browse more
    </a>
  </div>
</div>
`;
